Wireframe/css/style.css

```diff
@@ -0,0 +1,161 @@
+
+
+body{
+    background-color:#e5e0e0;
+    height: 100vh;
+    width: 100;   
+    
+}
+
+#index-h1 { 
+    font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: 30px;
+    font-weight: 600;
+    color: #9a9994;
+    text-transform: uppercase;
+    margin: 0 auto;
+    text-align: center;
+    padding: 40px 0;
+  
+
+}
+h1{
+    color:  #f14e32;
+    margin-left: 0px;
+}
+.first-p{
+    font-family: Arial, Helvetica, sans-serif;
+line-height: 30px;
+margin-left: 50px;
+color: #4e443c;
+    
+}
+
+#index-p{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    color: #4e443c;
+    text-transform: inherit;
+    margin: 0 auto;
+    text-align: center;
+    padding: 50px 0;
+}
+.git-logo{
+    display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 70px;
+    
+}
+.git-info{
+    display: block;
+    width: 1400px;
+    margin-left: 30px;
+    
+}
+.image{
+    display: block;
+   text-align: center;
+    margin-top: 50px;
+
+   
+}
+.first-p{
+    display:inline-block;
+  
+}
+h2{
+    color: #f14e32;
+    font-size: 40px;
+    margin-left: 30px;
+
+}
+
+
+.p1{
+    display: inline-block;
+    width: 500px;
+    vertical-align: top;
+    margin: 50px 90px 90px;
+    margin-top: 100px;
+    color: #4e443c;
+
+}
+.p2{
+    display: inline-block;
+    width: 500px;
+    margin-left: 90px;
+    margin-top: 100px;
+    margin-right: 90px;
+    color: #4e443c;
+  
+
+ 
+}
+.Paragraph{
```

Comment:
Best practice is not to capitalize class names. Please, update.

Reply:
Ok i will do